cmd/kuberhealthy/util.go

```diff
@@ -52,10 +52,12 @@ func notifyChanLimiter(maxSpeed time.Duration, inChan chan struct{}, outChan cha
 		log.Println("channel notify limiter witnessed an upstream message on inChan")
 		for {
 			select {
-			case <-time.After(maxSpeed):
-				outChan <- struct{}{}
+			//case <-time.After(maxSpeed):
+			//	outChan <- struct{}{}
 			case <-inChan:
 				log.Println("channel notify limiter witnessed an upstream message on inChan and is waiting an additional", maxSpeed, "before sending output")
+				time.After(maxSpeed)
+				outChan <- struct{}{}
```

Comment:
added the `time.After` here so that outChan is pinged only when inChan gets an actual KHCheck event and then waits 10 seconds before signaling to outChan